api/src/routes/anime/animeRoutes.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import { animeController } from "@@root/controllers";
+import { extractPaginationParams } from "@@root/middleware";
+
+const animeRoutes = express.Router();
+
+animeRoutes.route("/anime").get(extractPaginationParams, animeController.index);
```

Comment:
You don't need to put `/anime` here since that will be defined in `app.ts`.

Since in `app.ts` the routes import will look like 

```ts
...
// src/app.ts

import { animeRoutes } from './routes/';
...
app.use("/anime", animeRoutes);
...
```
The animeRoutes index file handles what to do after the `/anime`. My reason for this structure is to categorize routes associated with each other. 
For example, when we add route handlers for `AnimeCharacter` all we have to add is a `animeCharacterRoutes.ts` inside `src/routes/anime` directory. And then inside the `src/routes/anime/index.ts` we add `router.use("/characters", animeCharacterRoutes)`. 

```ts
// src/routes/anime/index.ts

router.use("/", animeRoutes); // handles `/anime`
router.use("/characters", animeCharacterRoutes); // handles `/anime/characters`
```